CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ endif()
 
 if(RAJA_ENABLE_HIP)
   message(STATUS "HIP version: ${hip_VERSION}")
+  set(RAJA_DEFAULT_WAVESIZE "64" CACHE STRING "Default wave size for GPU architecture. E.g. MI200/MI300 this is 64.")
```

Comment:
This should probably have HIP in the name.

Reply:
Switched it to RAJA_HIP_WAVESIZE - I'm not sure 'default' was needed.